src/main/resources/modelling-fields/census/fraction-active-transport-to-work-field.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Is this a duplicate of the field outlined [here](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/blob/master/src/main/resources/modelling-fields/city-indices/active-transport/ActiveTransportToWork-field.json)? Potentially under a different modelling field group?

Reply:
Yes, this is a duplicate. My idea was to move the re-usable sub-components of the indices into a thematic structure outside the city-indices directory. I.e. that all transport related things are together, all environment together, etc. If this is accepted, the active transport index could use the new component.

That said, going forward, I don't think that directory structure is sufficiently good organisation of model components and we should go for a tagging system along the lines described in #311.